render_bundle/render_bundle.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+"""Python script to generate one of the potential SD-CORE bundle variant."""
+
+import argparse
+import enum
+from typing import Optional
+
+from bundles import SDCoreUserPlane
+
+
+class SDCoreBundleVariant(enum.Enum):
+    """Possible bundles to generate."""
+
+    SDCORE_USER_PLANE = SDCoreUserPlane
+
+
+def _parse_args() -> tuple[bool, str, str, str]:
+    """Parses user provided arguments.
+
+    Returns:
```

Comment:
`channel` is missing from the docstring.

Reply:
Fixed